iOS/AppNavigator.swift

```diff
@@ -249,7 +295,13 @@ class AppNavigator {
         }
     }
 
-    func show(course: Course, with courseArea: CourseArea = .learnings) {
+    @available(iOS 13.0, *)
+    func presentInAnotherWindow(course: Course, courseArea: CourseArea, courseOpenAction: CourseOpenAction, courseClosedAction: CourseClosedAction) {
```

Comment:
Here, you are not considering `courseOpenAction` and `courseClosedAction`, which are mandatory for example when opening an item in a course.